docs/sources/enterprise/reporting.md

```diff
@@ -39,10 +39,45 @@ Currently only Organization Admins can create reports.
 1. Select the layout option for generated report: **Portrait** or **Landscape**.  
 1. Enter scheduling information. Options vary depending on the frequency you select.
 1. **Save** the report.
+1. **Send test mail** after saving the report to verify that the report looks like you expect it to.
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_create_new.png" max-width="500px" class="docs-image--no-shadow" >}}
 
-## Send test mail
+### Scheduling
+
+Scheduled reports can be sent on a weekly, daily, or hourly basis. You may also disable scheduling for when you either want to pause a report or send it via the API.
+
+All scheduling is approximative and indicates when the reporting service will start rendering the dashboard.
+
+#### Hourly
+
+Hourly reports are generated once per hour and takes two arguments:
```

Comment:
I am confused by your use of "arguments" here. Are these options in the UI that you fill in or select? If so, then they are probably fields. 

This is end user documentation, not developer notes.

Reply:
I feel as I cannot verbatim just replace “arguments” with “fields” here, but it makes perfect sense that arguments is something that isn’t used outside of the command-line/API terminology. I’ll dig through the docs a little and write up something hopefully more user-centric tomorrow.